spec/forms/normalizes_form_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     context 'when normalizing attributes with apply_to_nil: true' do
       it 'applies normalization to nil values and saves attributes' do
-        expect(nil).to receive(:strip).twice
+        allow(nil).to receive(:strip).twice
```

Comment:
will this still check that `strip` should be called twice?

Reply:
Yeah!

This code `allow(nil).to receive(:strip).once` returns error in this test.

Reply:
I think I prefer `expect` syntax more as I more want to see in tests what is expected, than I want to see mocks.